syft/formats.go

```diff
@@ -1,49 +0,0 @@
-package syft
-
-import (
-	"github.com/anchore/syft/syft/formats"
-	"github.com/anchore/syft/syft/formats/cyclonedxjson"
-	"github.com/anchore/syft/syft/formats/cyclonedxxml"
-	"github.com/anchore/syft/syft/formats/github"
-	"github.com/anchore/syft/syft/formats/spdxjson"
-	"github.com/anchore/syft/syft/formats/spdxtagvalue"
-	"github.com/anchore/syft/syft/formats/syftjson"
-	"github.com/anchore/syft/syft/formats/table"
-	"github.com/anchore/syft/syft/formats/template"
-	"github.com/anchore/syft/syft/formats/text"
-	"github.com/anchore/syft/syft/sbom"
-)
-
-// these have been exported for the benefit of API users
-// TODO: deprecated: now that the formats package has been moved to syft/formats, will be removed in v1.0.0
```

Comment:
It seems like these could still remain in place until 1.0, honoring the current promise from the deprecation notice

Reply:
There isn't a nuance, I'll add these back in. 👍